ogórek.go

```diff
@@ -238,6 +238,22 @@ func (d Decoder) Decode() (interface{}, error) {
 	return d.pop(), nil
 }
 
+func (d *Decoder) readLine() ([]byte, error) {
+	var has_more bool = true
```

Comment:
just call this isPrefix and assign to it in L245. You can predeclare `readData` and `err` as well. Also please use camelCase.